static/src/js/components/RevisionList/RevisionList.jsx

```diff
@@ -0,0 +1,251 @@
+import React, {
```

Comment:
There is a difference in feature compared to SIT
<img width="1604" alt="Screenshot 2024-04-08 at 8 55 28 AM" src="https://github.com/nasa/mmt/assets/63017465/3da07128-df18-4fbe-accc-36d6782f43d3">

Reply:
What is the difference that you're referring to? There are many differences but many are known and expected.

Reply:
SIT MMT has  breadcrumbs, Revision Date's format is different